ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master__upgrade.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
```

Comment:
Why choosing 4.14? How about choosing 4.16 as the first PR? You can change your file name to ...openshift-cert-manager-operator-master__operator-upgrade-in-4.16.yaml so that future you can copy it to 4.17 et al.

Reply:
The intention here is just for PR rehearsal. (As of now, OCP 4.14 has the longest upgrade path: 1.11 -> 1.12 -> 1.13 -> 1.14.) If this path can pass successfully, we will have more confidence in changing the "INSTALL_CSV" later to whatever version we prefer.

Reply:
OK. But you can still change the config yaml file name as above said.